index.html

```diff
@@ -43,6 +45,8 @@
             </ul>
         </nav>
     </div>
+    <div id="mobile-menu">
+    </div>

```

Comment:
- [ ] Kindly ensure your design resembles the given design. Please add the underline below each nav link as seen in the main design.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/131528701-6e6acbd5-bffe-43f3-8f02-4d4aa5ea09b8.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/131528788-e6ac45d5-5acd-4d73-88cb-d93aad0f8d06.png' height='250px'> |

Reply:
![Screenshot from 2021-08-31 16-37-45](https://user-images.githubusercontent.com/58692590/131532882-751dc35e-c328-434c-902c-3316a4be22c0.png)
Thank you.
Issue Fixed

Reply:
This issue is well resolved! ✔️